src/java.base/share/classes/java/net/NetworkInterface.java

```diff
@@ -271,6 +308,16 @@ public static NetworkInterface getByName(String name) throws SocketException {
     /**
      * Get a network interface given its index.
      *
+     * @apiNote
+     * The returned interface instance may reflect a snapshot of the
+     * configuration taken at the time the instance is created.
+     * To see updates to the configuration may, or may not, require
```

Comment:
The sentence "To see updates ..." is hard to read. How about "The network configuration may change at any time, this method may need to be invoked to obtain a more up to date view of the network interface".